enterprise/internal/campaigns/store_campaign_specs_test.go

```diff
@@ -234,4 +234,74 @@ func testStoreCampaignSpecs(t *testing.T, ctx context.Context, s *Store, _ repos
 			}
 		}
 	})
+
+	t.Run("DeleteExpiredCampaignSpecs", func(t *testing.T) {
+		tests := []struct {
+			createdAt   time.Time
+			hasCampaign bool
+			wantDeleted bool
+		}{
+			{
+				hasCampaign: false,
+				createdAt:   clock.now(),
+				wantDeleted: false,
+			},
+			{
+				hasCampaign: false,
+				createdAt:   clock.now().Add(-8 * 24 * time.Hour),
```

Comment:
Maybe to deal with any future TTL changes (untested):

```suggestion
				createdAt:   clock.now().Add(-cmpgn.CampaignSpecTTL - 24 * time.Hour),
```

Reply:
Ahh! Thanks for that. I forgot to update those after moving the constant.